cypress.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  extends: ['plugin:cypress/recommended'],
+  rules: {
+    'import/no-extraneous-dependencies': [
+      'error',
+      {
+        devDependencies: ['**/*.cy.{js,ts,tsx}'],
```

Comment:
I think this overrides the devDependencies in es6. Or does it get merged with it?

I think we want to put this `**/*.{cy,spec,stories,test}.*` for the devDependencies for this rule in es6. 

And I was thinking that we should add '.storybook/**' and 'scripts/**' to it too. since all of these are going to be devDeps.

Reply:
not sure if there is a significant difference in doing a specific set of extensions or not:
`'**/*.{cy,spec,stories,test}.[tj]s?(x)'` vs `'**/*.{cy,spec,stories,test}.*'`

Reply:
Most repos will probably not be linting the .storybook directory so the devDep won't matter for them, but for the few that do, we will want .storybook directory to be as a devDep for the rule.
eslint, by default, does not lint . directories.